daq/runner.py

```diff
@@ -115,9 +115,9 @@ def get_run_info(self):
             'uname': self._sys_uname,
             'daq_run_id': str(self._daq_run_id)
         }
-        data_retention_days = float(self.config.get('run_data_retention_days'))
+        data_retention_days = self.config.get('run_data_retention_days', None)
```

Comment:
There's no actual default here -- should be some retention value.  Behavior should be initially similar to what it was before.